webview/src/experiments/components/table/CellHintTooltip.tsx

```diff
@@ -0,0 +1,21 @@
+import React, { ReactNode, ReactElement } from 'react'
+import styles from './styles.module.scss'
+import Tooltip, {
+  CELL_TOOLTIP_DELAY
+} from '../../../shared/components/tooltip/Tooltip'
+
+export const CellHintTooltip: React.FC<{
+  tooltipContent: ReactNode
+  children: ReactElement
+}> = ({ tooltipContent, children }) => {
+  return (
+    <Tooltip
```

Comment:
[Q] Do we want to move the indicator tooltips over to this?

Reply:
> [Q] Do we want to move the indicator tooltips over to this?

Not quite sure what you mean 🤔 Could you expand a bit more on the question?

Reply:
We have two sizes of tooltip hints in the table. Do we want the tooltips for the indicators in the top left-hand corner to be this size?

https://user-images.githubusercontent.com/37993418/189995145-5b086ca8-73e7-41fc-a502-a73c4c5ee2f9.mov

Reply:
I don't have a preference 🤔 cc @maxagin

Reply:
Just to clarify.
We are talking about a tooltip text when the mouse moves over the element. Right?

![Untitled-1](https://user-images.githubusercontent.com/98249521/190280419-ae4851cb-d542-487e-93c2-fc2e0384f826.jpg)

If yes. Is there a reason we use different sizes?

Reply:
The placement of the tooltip in relation to the cursor is a bit different from the context menu placement. Here are examples.

<img width="1007" alt="Screen Shot 2022-09-14 at 7 38 51 PM" src="https://user-images.githubusercontent.com/98249521/190281233-7dd648f1-bdb8-4545-92b5-042516d587e6.png">

Reply:
> We have two sizes of tooltip hints in the table. Do we want the tooltips for the indicators in the top left-hand corner to be this size?

I see now what @mattseddon talking about -font size. I would make all the tooltips the same as what we have on the table now if possible + adjust the placement.